src/api/app/services/workflows/scm_event_subscription.rb

```diff
@@ -0,0 +1,25 @@
+module Workflows
+  class ScmEventSubscription
```

Comment:
```suggestion
  class ScmEventSubscriptionCreator
```

I think we should name services for what they do....